Libraries/Components/ScrollView/ScrollView.js

```diff
@@ -649,9 +649,6 @@ const ScrollView = React.createClass({
         {...contentSizeChangeProps}
         ref={this._setInnerViewRef}
         style={contentContainerStyle}
-        removeClippedSubviews={
-          hasStickyHeaders && Platform.OS === 'android' ? false : this.props.removeClippedSubviews
-        }
```

Comment:
this should be `removeClippedSubviews={this.props.removeClippedSubviews}` otherwise ScrollView will never remove clipped subviews